src/main/java/org/broadinstitute/hellbender/engine/ReadsDataSource.java

```diff
@@ -97,7 +98,8 @@ public ReadsDataSource( final File samFile, SamReaderFactory customSamReaderFact
      * Initialize this data source with multiple SAM/BAM files and a custom SamReaderFactory
      *
      * @param samFiles SAM/BAM files, not null.
-     * @param customSamReaderFactory SamReaderFactory to use, if null a default factory with validation stringency SILENT is used.
+     * @param customSamReaderFactory SamReaderFactory to use, if null a default factory with no reference and validation
+     *                               stringency SILENT is used.
```

Comment:
Makes sense. I'll do that in part 2 of this submission (when I do the Spark tools).